python/docs/source/reference/pyspark.testing.rst

```diff
@@ -0,0 +1,28 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
You should also probably add this to `python/docs/source/reference/index.rst`

Reply:
@HyukjinKwon Is there a way to build docs locally? Just to check what it looks like.